.circleci/config.yml

```diff
@@ -233,6 +241,9 @@ workflows:
       - unit_test_browser:
           name: unit_test_browser_sayt_driver_plugin
           <<: *requires_build_final
+      - unit_test_browser:
```

Comment:
Does this need to be added to the `unt_test` jobs section as well?

Reply:
Yes